avocado/utils/asset.py

```diff
@@ -58,7 +59,7 @@ class Asset:
     """
 
     def __init__(self, name, asset_hash, algorithm, locations, cache_dirs,
-                 expire=None):
+                 expire=None, meatadata=None):
```

Comment:
Typo: s/meat/meta/, unless you intended for some "beefy" data :)

Reply:
Nice catch, this is a really funny typo :smile: